src/frontend/src/components/Type/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+interface Props {
+  expungement_result: any;
+}
+
+export default class Type extends React.Component<Props> {
+  render() {
+    const {
+      type_eligibility,
+      type_eligibility_reason,
+      time_eligibility
+    } = this.props.expungement_result;
+
+    const eligible = (
+      <div className="relative mb3">
+        <i
+          aria-hidden="true"
+          className="absolute fas fa-check-circle green"
+        ></i>
+        <div className="ml3 pl1">
+          <span className="fw7">Type:</span> Eligible{' '}
+          <span className="nowrap">{type_eligibility_reason}</span>
```

Comment:
```suggestion
          <span className="nowrap">({type_eligibility_reason})</span>
```

Otherwise we will get messages like "Eligible Eligible under 137.225(1)(b)"